src/basic/sysctl-util.c

```diff
@@ -96,6 +96,26 @@ int sysctl_write_ip_property(int af, const char *ifname, const char *property, c
         return sysctl_write(p, value);
 }
 
+int sysctl_write_ip_neigh_property(int af, const char *ifname, const char *property, const char *value) {
```

Comment:
Please do not abbreviate the name. `neigh` -> `neighbor`

Reply:
Done.